media/js/chat.js

```diff
@@ -0,0 +1,4 @@
+window.client = new window.LCB.Client();
```

Comment:
I removed it from chat.html

@geoffwhittington was saying that inline JavaScript impacts Content Security Policy